### DISCUSSION TITLE
Batman Arkham Origins

### DISCUSSION THREAD

**Discussion by HexPa5** [EliotVU/Unreal-Library#86] [Q&A]
Hello, we actually working at upscaling textures for arkham origins with other tools and converters, and we are able to path that in TFC texture packages but I need to open Engine.u and Bmgame.u without any issues to find out the texture specular pixel size and game characters textures pixel size too, because we had issues running the game and launching on that upscaled textures and I need to bypass typically the pixel size by finding in these classes, objects definition of texture pixels to increase that value in Hex using UE Explorer. 
So it's possible to do that?

**Comment by EliotVU** [comment]
I think it's possible to change the texture / pixel size properties, but to actually use a bigger texture mip is not quite possible with UELib :/ Isn't this something UPK Explorer is specialized in?
